Dockerfile.common

```diff
@@ -89,21 +90,25 @@ RUN cd /girder                             \
 ADD gaia                /girder/gaia
 ADD minerva             /girder-components/plugins/minerva
 ADD database_assetstore /girder-components/plugins/database_assetstore
-ADD girder_ktile        /girder-components/plugins/girder_ktile
+ADD large_image        /girder-components/plugins/large_image
 ADD ansible/bsve        /girder-components/plugins/bsve
 RUN girder-install plugin -f /girder-components/plugins/database_assetstore
-RUN girder-install plugin -f /girder-components/plugins/girder_ktile
+RUN girder-install plugin -f /girder-components/plugins/large_image
 RUN girder-install plugin -f /girder-components/plugins/bsve
 RUN girder-install plugin -f /girder-components/plugins/minerva
 
 RUN pip install -e /girder/gaia                             \
  && pip install -r /girder/gaia/requirements.txt            \
  && pip install -e /girder-components/plugins/minerva
 
+RUN cd /girder-components/plugins/large_image          \
+ && python setup.py install || python setup.py install \
```

Comment:
@matthewma7 Can you explain this line.

Reply:
@dorukozturk Good catch, I copied this from another project and didn't notice this. But I think I am able to replace this a different method. Please take a look at the latest state.